res/css/views/elements/_GenericEventListSummary.pcss

```diff
@@ -21,6 +21,16 @@ limitations under the License.
         margin-right: $spacing-8;
     }
 
+    .mx_GenericEventListSummary_spacer {
+        border-bottom: 1px solid $primary-hairline-color;
+        margin-left: 63px; /* TODO: Use a varilable */
+        line-height: $font-30px; /* TODO: Use a varilable */
+
+        .mx_IRCLayout & {
+            line-height: var(--line-height);
```

Comment:
This declaration has the expected line-height applied to the spacer line against `line-height: $font-30px;`.